druxxy.info.yml

```diff
@@ -22,7 +22,6 @@ dependencies:
   - custom_pub
   - datetime
   - datetime_range
-  - dblog
```

Comment:
it makes initial install without logs, so we should add monolog instead (better preconfigured)

Moreover it will break CI https://github.com/skilld-labs/skilld-docker-container/blob/master/.gitlab-ci.yml#L505